login/token_context/token.go

```diff
@@ -0,0 +1,31 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
+
+import (
+	"context"
+)
+
+type contextTMKey int
+
+const (
+	//contextTokenManagerKey is a key that will be used to put and to get `tokenManager` from goa.context
+	contextTokenManagerKey contextTMKey = iota + 1
+)
+
+//ReadTokenManagerFromContext returns tokenManager from context.
+// Must have been set by ContextWithTokenManager ONLY
+func ReadTokenManagerFromContext(ctx context.Context) interface{} {
```

Comment:
I see... But let's at least mention in the function description that the returned interface can be safely converted to token.Manager if not nil.